docs/content/docs/plugins/BuildChecks.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Build checks plugin
+type: docs
+---
+
+# Build checks plugin
+
+This plugin verifies [common build problems]({{< relref "#checks">}}) with environment and project configuration.
+
+## Getting started
+
+Apply the plugin in the root `build.gradle` file:
+
+```groovy
+plugins {
+    id("com.avito.android.buildchecks")
+}
+
+buildChecks {
+    enableByDefault = true
```

Comment:
can we set it `true` by default and not mention in how-to doc?

setting `false` makes sense only in tests, so not a user facing option at all

Reply:
I'll try. The problem here is backward compatibility only.

Reply:
Fixed.
Luckily, enabled by default become simpler in implementation also.
For backward compatibility, the plugin doesn't use `buildCheck` if we have legacy properties.